code/modules/medical/computer/cloning.dm

```diff
@@ -92,7 +93,8 @@
 				to_chat(user, "You insert \the [W].")
 				src.updateUsrDialog()
 				return 1
-	return
+    if (istype(W,/obj/item/weapon/card/emag))
```

Comment:
don't listen to @PJB3005 -  add a machine flag EMAGGABLE and make
/obj/machinery/computer/cloning/emag()
emagged = 1